SofaKernel/framework/sofa/defaulttype/DataTypeInfo.h

```diff
@@ -139,6 +142,8 @@ struct DataTypeInfo
     {
     }
 
+    // mtournier: wtf is this supposed to do?
```

Comment:
This kind of comment are useless. 
Drop them or maybe use a more formal approach so that we really get an answer of the use of this function or the deprecation the weird function.

Reply:
Sorry, those are my own personal comments that were not meant to make it in the final PR.

Reply:
You often show your surprise with wtf comments which I found very funny because I have the same but I don't write them :)